pubspec.yaml

```diff
@@ -10,6 +10,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+  shared_preferences: ^2.0.15
```

Comment:
Why was this added? Can't seem to find references in the changed code for this

Reply:
Oh, this should only be added in the example project. my bad